pyomo/solvers/plugins/solvers/CPLEX.py

```diff
@@ -546,6 +559,10 @@ def process_logfile(self):
             elif len(tokens) >= 3 and tokens[0] == "Variables":
                 if results.problem.number_of_variables is None: # CPLEX 11.2 and subsequent versions have two Variables sections in the log file output.
                     results.problem.number_of_variables = int(tokens[2])
+                if len(tokens) >= 5 and "Nneg" in tokens[3]:
+                    results.problem.number_of_continuous_variables = int(tokens[4][:-1])  # remove trailing comma
```

Comment:
Might be clearer to do `.rstrip(',')` and that way you don't need to comment.

Reply:
done